### PR TITLE
feat: implement socket, define socket event constants

### PR DESCRIPTION
**Pull request by TMNhat1810**

### CODE REVIEW

File: app/socket/socket.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    socket.on(SOCKET_EVENT.ISSUE.DESELECT, (data) => {
+      io.to(socket.roomId).emit(SOCKET_EVENT.ISSUE.DESELECT, data);
+    });
+
+    socket.on('test', (data) => {
+      const ret = {
+        user: socket.userId,
+        room: socket.roomId,
+      };
+      console.log(data);
```
Comment (ces-anduynguyen) [inline app/socket/socket.js:73]: remove console.log

File: app/socket/socket_event.js
Comment (ces-anduynguyen) [inline app/socket/socket_event.js:1]: should put this file in `constants` folder?

File: app/socket/socket.js (hunk 1 above)
Comment (TMNhat1810) [reply app/socket/socket.js:73]: oh no. my bad

File: app/socket/socket_event.js
Comment (TMNhat1810) [reply app/socket/socket_event.js:1]: got it!

**Review by ces-hoangdao — APPROVED** [review]

**Pull request merged**